src/data/faq.json

```diff
@@ -968,15 +968,13 @@
                         "anchor": "encounter_count_19",
                         "active": true,
                         "textblock": [
-                            "With the introduction of Corona-Warn-App version 1.9, a new version of the Exposure Notification Framework (ENF) from Apple/Google is used, version 2.0. The data from this interface is used as the basis for calculating the individual risk.",
-                            "Previously, the number of encounters displayed in the app has not been filtered. Up to version 1.7.1, encounters with very short contact durations (a few seconds were sufficient) were also counted and displayed in the app. However, due to the stored criteria for the risk calculation, these were only displayed as low-risk encounters.",
-                            "<b>In simplified terms:</b> Under Exposure Notification Version 2.0, the operating system also logs encounters with a risk lower than 'low risk' (green). However, since these encounters are not relevant from a virological perspective, the Corona-Warn-App filters them out.",
-                            "<b>In detail:</b>",
-                            "With the interface's new version, it is now possible to control more precisely which encounters should be counted. However, in the operating system's contact log, a count is performed. However, this is not an error.",
+                            "With the introduction of Corona-Warn-App version 1.9, a new version of the Exposure Notification Framework (ENF) from Apple/Google is used, version 2.0  (also known as ExposureWindow mode by Google). The data from this interface is used as the basis for calculating the individual risk.",
+                            "Up to version 1.7.1, the number of encounters displayed in the app has not been filtered. Encounters with very short contact durations (a few seconds were sufficient) were also counted and displayed in the app. However, due to the stored criteria for the risk calculation, these were only displayed as low-risk encounters.",

```

Comment:
"Up to version 1.7.1, the number of encounters displayed in the app has not been filtered."

=> 

"Up to version 1.7.1, the numbers of encounters displayed in the app were not filtered."

... since 1.7.1 is now further in the past (and I changed number to numbers to make the grammar easier!)

Reply:
Many thanks !